cosmwasm/utils.js

```diff
@@ -66,11 +66,7 @@ const getChains = (config, { chainNames, instantiate2 }) => {
         throw new Error('Cannot pass --instantiate2 with more than one chain');
     }
 
-    const undefinedChain = chains.find((chain) => !config.chains[chain.toLowerCase()] && chain !== 'none');
-
-    if (undefinedChain) {
-        throw new Error(`Chain ${undefinedChain} is not defined in the info file`);
-    }
+    chains.every((chain) => chain === 'none' || getChainConfig(config, chain));
```

Comment:
this value doesn't seem to be used anywhere

Reply:
`getChainConfig` throws an error if not found